validator_types/src/lib.rs

```diff
@@ -81,6 +87,7 @@ impl Validator {
             Validator::Regex(_) => "regex",
             Validator::Range { .. } => "range",
             Validator::Length { .. } => "length",
+            Validator::LengthUTF16 { .. } => "length_utf16",
```

Comment:
```suggestion
            Validator::LengthUtf16 { .. } => "length_utf16",
```

For consistency with Rust code style, you might want to use `Utf` in identifiers.

E.g. https://doc.rust-lang.org/std/str/struct.EncodeUtf16.html

Reply:
I'll address code style suggestions if the approach with an extra builtin validator type is desired for the crate users.

We could gather more comments/thumbs up in the MR description for more feedback.